sto/cli/main.py

```diff
@@ -601,6 +601,51 @@ def version(config: BoardCommmadConfiguration):
     print(config.version)
 
 
+@cli.command(name="kyc-deploy")
+@click.pass_obj
+def kyc_deploy(config: BoardCommmadConfiguration):
+    """
+    Deploys Kyc contract to desired ethereum network
+    network, ethereum-abi-file, ethereum-private-key, ethereum-node-url are required args
+    """
+    from sto.ethereum.utils import deploy_contract
+    deploy_contract(config, contract_name='BasicKYC')
```

Comment:
We can layer change this to deploy with you set `--transfer-mode=restricted` when doing an issuance.